pkg/lldp/client.go

```diff
@@ -92,11 +92,13 @@ func (l *Client) Close() {
 func (l *Client) Neighbors(neighChan chan Neighbor) {
 	for {
 		for packet := range l.Source.Packets() {
+			//nolint
```

Comment:
Since we only take care of layers.LayerTypeEthernet wouldn't it be more readable to use something like this?
```
if packet.LinkLayer().LayerType() != layers.LayerTypeEthernet {
    continue
}
...
```